en/docs/guides/identity-verification/idvp-ui.md

```diff
@@ -0,0 +1,307 @@
+# Onboarding the UI of an IDV Provider to the Identity Server
+
+The UI of an IDV provider is a combination of static and dynamically generated sections. The static sections are already baked into the UI implementation while the metadata for the dynamic sections is pulled from the extensions implementation in the Identity Server [git issue].
```

Comment:
```suggestion
The UI of an IDV provider is a combination of static and dynamically generated sections. The static sections are already baked into the UI implementation, while the metadata for the dynamic sections is pulled from the extensions implementation in the Identity Server [git issue].
```

Reply:
should we link a [git issue] here? don't understand why we have added it.

Reply:
No it is not required. I have removed that. @nipunsampath Is there any doc link to add here for the extensions implementation?